### ISSUE TITLE
Modify workflow.py to allow skipping the copy step

### ISSUE THREAD

**Issue by theferrit32** [clingen-data-model/clinvar-ingest#178]
For re-running weekly releases, the file is eventually deleted from clinvar's ftp site, but we have the copy in GCS already. We should be able to easily re-run the workflow on the file in GCS without needing to do the copy.

Can auto-detect that the copy step should be skipped from the input values. Two options:
- we can accept different input values for the host, like `gs://<bucket-name>`, which would indicate that the file is already in a bucket, not on an ftp/http site, and if the bucket name matches the bucket we are copying to, skip the copy.
- accept the same input values, like:
    ```
    host="https://ftp.ncbi.nlm.nih.gov"
    directory="/pub/clinvar/xml/VCV_xml_old_format"
    name="ClinVarVariationRelease_2024-02.xml.gz"
    size=3298023159
    last_modified="2024-01-07T15:47:16"
    released="2024-02-01T15:47:16"
    release_date="2024-02-01"
    ```
    but when we construct the `copy` destination location (e.g. `gs://bucket/path/ClinVarVariationRelease_2024-02.xml.gz`) we can check if it already exists and skip the `copy` if so, which both satisfies #63 and makes it so the workflow doesn't break for re-running past runs, by omitting the `copy` step.

**Comment by theferrit32** [comment]
This is pretty high priority actually because we can't re-run on a past file like `https://ftp.ncbi.nlm.nih.gov/pub/clinvar/xml/VCV_xml_old_format/weekly_release/ClinVarVariationRelease_2024-0603.xml.gz` since it no longer exists.

**Comment by theferrit32** [comment]
Closed by #181 

See https://github.com/clingen-data-model/clinvar-ingest/blob/0c842d8acf0b333f33f30573bbd6b6604a307b95/misc/bin/execute-job.sh#L66-L73

 as an example of input args
